src/toplevel_generators.jl

```diff
@@ -158,13 +164,15 @@ function generate_wrapper_load(src_name, pkg_uuid, __source__)
                 end
 
                 # From the available options, choose the best wrapper script
-                select_platform(valid_wrappers)
+                platform = augment_platform(HostPlatform())
+                select_platform(valid_wrappers, platform)
             end
         end
 
         # Load in the wrapper, if it's not `nothing`!
         if best_wrapper === nothing
-            @debug(string("Unable to load ", $(src_name), "; unsupported platform ", triplet(HostPlatform())))
+            platform = augment_platform(HostPlatform())
+            @debug(string("Unable to load ", $(src_name), "; unsupported platform ", repr(platform)))
```

Comment:
```suggestion
            @debug(string("Unable to load ", $(src_name), "; unsupported platform ", triplet(platform)))
```

Looks like triplet is a misnomer? https://github.com/JuliaPackaging/BinaryBuilder.jl/blob/35c373a07418f86472455beb5ccd4775d50022d4/src/AutoBuild.jl#L1158 and it contains the tags? 

https://github.com/JuliaBinaryWrappers/Enzyme_jll.jl/tree/main/src/wrappers

Reply:
Yes, that's right.  `triplet()` is [increasingly inaccurately named](https://www.parigibooks.com/pictures/30998.jpg) these days.